artifacts/deploy/karmada-metrics-adapter.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     matchLabels:
       app: karmada-metrics-adapter
       apiserver: "true"
-  replicas: 1
+  replicas: 2
```

Comment:
/cc @XiShanYongYe-Chang 
We are unable to deploy the metrics adapter with 2 replicas due to insufficient resources. Should we proceed with only 1 replica?

Reply:
Considering that it is a dual-active component, I think we can deploy only one instance in CI. The issue you mentioned also reveals that when we add new components later on, we may still encounter resource shortages. At that time, it may be necessary to consider adding a "kind node" node in CI.

Reply:
/cc @calvin0327

Reply:
ok, I agress with you.